### PR TITLE
EW refs #44850 Multi Step page view

### PR DESCRIPTION
**Pull request by viktoriia-sobutska**
- Included Bootstrap scss & js files.
Created a new variant for Tabs pattern - Pills with button option.

- Created template for Multi step page and included tabs pattern.
- Added to pattern styles. Added at the bottom bottoms. Added js for switching.
- Added to right arrow svg.
Added variable variation for link anchor section.
Added summery field into template.
Got from preprocess function title and field_sequential_non_sequential.
Installed twig_field_value.
Added variables into tailwind config file.

- Created a new pattern Titled content.
Added field related content to it. Added conditions is isset content.
Added generals styles.

### CODE REVIEW

**Review by alt-ew — CHANGES_REQUESTED** [review]

File: docroot/themes/custom/yukonca_glider/tailwind.config.js [hunk 1]
```diff
@@ -62,24 +65,29 @@ module.exports = {
...
         h5:             ['1.188rem', '1.5rem'],
         'base-mobile':  ['0.938rem', '1.375rem'],
-        'heading': fontSize(22, 24),
+        heading: fontSize(22, 24),
         'default-base': ['1rem', '1.25rem'],
-        'grande':       ['1.25rem', '1.625rem'],
-        'legendes':     ['0.813rem', '1.25rem'],
+        grande:       ['1.25rem', '1.625rem'],
+        legendes:     ['0.813rem', '1.25rem'],
+        circles:      ['2rem', '2rem'],
```
[the remark this answers is not in the document]
Comment (viktoriia-sobutska) [reply docroot/themes/custom/yukonca_glider/tailwind.config.js:72]: It was removed

File: docroot/themes/custom/yukonca_glider/tailwind.config.js [hunk 2]
```diff
@@ -99,6 +107,11 @@ module.exports = {
       '2xl'   : '1440px',
       '3xl'   : '1600px',
     },
+    borderWidth: {
+      DEFAULT: '1px',
```
[the remark this answers is not in the document]
Comment (viktoriia-sobutska) [reply docroot/themes/custom/yukonca_glider/tailwind.config.js:111]: removed

File: docroot/themes/custom/yukonca_glider/patterns/atoms/titled_content/pattern-titled-content.html.twig [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{% set prefix = 'titled-content' %}
+{% set classes = [
+  prefix,
+  prefix ~ '--' ~ variant,
+] %}
+
+<div {{ attribs.addClass(classes) }}>
+  <div class="border-solid border-t border-b-gray-lightgray my-7"></div>
```
Comment (viktoriia-sobutska) [inline docroot/themes/custom/yukonca_glider/patterns/atoms/titled_content/pattern-titled-content.html.twig:8]: figure it out https://tailwindcss.com/docs/border-width#using-without-preflight

File: docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+.node--type-multi-step-page {
```
[the remark this answers is not in the document]
Comment (viktoriia-sobutska) [reply docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss:1]: We need for general styles for this page

File: docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/pattern-tabs--variant-pills.html.twig [hunk 5]
```diff
@@ -0,0 +1,71 @@
+{# Prepare attributes #}
+{% set prefix = 'pills' %}
+{% set classes = [
+  prefix,
+  prefix ~ '--' ~ variant,
+  more_link|render ? 'has-more-link' : ''
+] %}
+
+{% set tab_id = title|trim|clean_id %}
```
[the remark this answers is not in the document]
Comment (viktoriia-sobutska) [reply docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/pattern-tabs--variant-pills.html.twig]: Changed it in preprocess function

File: docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/scss/styles.scss [hunk 6]
```diff
@@ -1 +1,64 @@
...
+      @apply bg-none;
+
+      &::before {
+        @apply bg-blue-200;
+      }
+
+      .pills-number {
+        @apply bg-blue-200;
+      }
+    }
```
Comment (alt-ew) [inline docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/scss/styles.scss:36]: More likely that we can move this part to the twig file.

File: docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/scss/styles.scss [hunk 7]
```diff
@@ -1 +1,64 @@
...
+  .tab-pane.fade {
+    @apply hidden;
+  }
+
+  .tab-pane.fade.show.active {
+    @apply block;
+
+    &:focus-visible {
+
+    }
```
Comment (alt-ew) [inline docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/scss/styles.scss:54]: Looks like we don't need this part.

File: docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/scss/styles.scss [hunk 8]
```diff
@@ -1 +1,64 @@
...
+  }
+}
+
+.pills-arrow {
+  .btn-links.hide {
+    .pills-btn-link {
+      display: none;
```
Comment (alt-ew) [inline docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/scss/styles.scss:61]: I think that tailwind has a class for this property.

File: docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/scss/styles.scss [hunk 9]
```diff
@@ -1 +1,64 @@
-/* Add your code here */
+.nav-pills {
+  &.sequential .nav-link {
+    &::before {
+      content: counter(my-awesome-counter);
```
[the remark this answers is not in the document]
Comment (alt-ew) [reply docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/scss/styles.scss:4]: Can we use a tailwind class here?

File: docroot/themes/custom/yukonca_glider/src/sass/base/_base.scss [hunk 10]
```diff
@@ -1,12 +1,43 @@
...
+    border-width: 0;
+    border-style: solid;
+  }
+
   html,
   body {
     @apply min-h-full font-base text-black;
   }
 
+  .field__content {
```
Comment (alt-ew) [inline docroot/themes/custom/yukonca_glider/src/sass/base/_base.scss:14]: Are you sure that we need to add these rules to all pages? 
This is a question to all changes in this file.

File: docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss (hunk 4 above)
[the remark this answers is not in the document]
Comment (alt-ew) [reply docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss:1]: Could you please check whether we need this margin in all nodes or now? If this rule is used in different content types then it would be good to make this rule more global.

File: docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+  .field__item {
+    @apply mb-5;
+  }
+}
+
+.nav-pills.non-sequential {
```
Comment (alt-ew) [inline docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss:7]: Looks like this change is not related to the multi-step page.

File: docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss [hunk 12]
```diff
@@ -0,0 +1,15 @@
...
+  .nav-link {
+    &::before {
+      @extend %icon;
+      @include getIcon("chevron-right");
+      @apply absolute p-0 text-[2rem];
```
Comment (alt-ew) [inline docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss:12]: Shouldn't we add 2rem font size to the tailwind config file?

File: docroot/themes/custom/yukonca_glider/tailwind.config.js [hunk 13]
```diff
@@ -51,53 +54,63 @@ module.exports = {
...
-      '3xl'   : '1600px',
+      xs: '576px',
+      sm: '667px',
+      md: '768px',
+      lg: '992px',
+      xl: '1200px',
+      '2xl': '1440px',
+      '3xl': '1600px',
+    },
+    listStyleType: {
```
Comment (alt-ew) [inline docroot/themes/custom/yukonca_glider/tailwind.config.js:110]: Why do we need this config here? This is a default tailwind rule and you don't change anything in this particular object.

**Review by alt-ew — CHANGES_REQUESTED** [review]

File: docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/js/tabs.js [hunk 14]
```diff
@@ -6,14 +6,14 @@
...
  */
 
-(function ($, Drupal) {
+(function ($, Drupal, once) {
   /**
    * Main script for the current theme
    */
   Drupal.behaviors.tabsJS = {
     attach (context) {
       const tabContainer = $('.tabs', context);
```
[the remark this answers is not in the document]
Comment (viktoriia-sobutska) [reply docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/js/tabs.js:15]: it is not working for Pills

File: docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss (hunk 12 above)
Comment (viktoriia-sobutska) [reply docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss:12]: we use it only in 1 place

File: docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/tabs.patterns.yml [hunk 15]
```diff
@@ -35,4 +52,4 @@ tabs:
           component:
             dist/css/styles.min.css: { minified: true }
         js:
-          dist/js/tabs.min.js: { minified: true }
+            dist/js/tabs.min.js: { minified: true }
```
Comment (viktoriia-sobutska) [inline docroot/themes/custom/yukonca_glider/patterns/organisms/tabs/tabs.patterns.yml:55]:
```suggestion
          dist/js/tabs.min.js: { minified: true }
```

**Review by alt-ew — APPROVED** [review]
